beacon-chain/powchain/engine-api-client/v1/auth.go

```diff
@@ -0,0 +1,32 @@
+package v1
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+	"github.com/pkg/errors"
+)
+
+// Implements the http.RoundTripper interface to add JWT authentication
+// support to an HTTP client used for interacting with an execution node.
+// See the specification for more details on the supported JWT claims:
+// https://github.com/ethereum/execution-apis.
```

Comment:
Do you have a better link so readers don't have to dig around that repo? I couldn't immediately find it